.github/workflows/mkdocs-deploy.yml

```diff
@@ -0,0 +1,32 @@
+# GCLINE@ 23NOV2020
```

Comment:
guess this should be amazon's copy right or leave empty :D

Reply:
I meant that more as a "who wrote this" but I can see the confusion, I'll fix